src/uct/ib/base/ib_iface.c

```diff
@@ -93,7 +93,7 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "enough, such as of atomic operations and small reads, will be received inline.",
    ucs_offsetof(uct_ib_iface_config_t, inl[UCT_IB_DIR_TX]), UCS_CONFIG_TYPE_MEMUNITS},
 
-  {"TX_MIN_SGE", "4",
+  {"TX_MIN_SGE", "5",
```

Comment:
It does not increase it, because with 5 sges it becomes the same size (80) as inline section here:
https://github.com/openucx/ucx/blob/master/src/uct/ib/mlx5/dv/ib_mlx5_dv.c#L85

Reply:
ok